posts/2020-09-01-jakarta-microprofile-beta-20.0.0.10.adoc

```diff
@@ -0,0 +1,336 @@
+}
+...
+----
+
+To view the status of each health check, access the either the `http://<hostname>:<port>/health/live` or `http://<hostname>:<port>/health/ready endpoints`.
+
+More information: 
+* link:https://github.com/eclipse/microprofile-health/releases/tag/3.0-RC3[MicroProfile Health Check 3.0 Release Page]
+* link:https://github.com/eclipse/microprofile-health/blob/master/spec/src/main/asciidoc/release_notes.asciidoc[Release notes]
+
+[#opentracing]
+=== MicroProfile OpenTracing 2.0
+
+MicroProfile OpenTracing 2.0 can be used to profile and monitor applications built using microservice architecture.
+
+MicroProfile OpenTracing 2.0 has upgraded the OpenTracing API to version 0.33.0.  This allows the 
+use of tracing backends and their libraries that are built on OpenTracing API 0.33.0.
+
+==== Try it now
+
+Include the following in the server.xml:
+
+[source, xml]
+----
+    <feature>mpOpenTracing-2.0</feature>
+----
+
+Also configure a tracing backend such as Jaeger or Zipkin 
+    For Jaeger, add the following maven dependencies in the application's pom.xml.
+
+[source, xml]
+----
+<dependency>
+    <groupId>io.jaegertracing</groupId>
+    <artifactId>jaeger-client</artifactId>
+    <version>1.2.0</version>
+</dependency>
+<dependency>
+    <groupId>org.slf4j</groupId>
+    <artifactId>slf4j-api</artifactId>
+    <version>1.7.30</version>
+</dependency>
+<dependency>
+    <groupId>org.slf4j</groupId>
+    <artifactId>slf4j-jdk14</artifactId>
+    <version>1.7.30</version>
+</dependency>
+----
+
+You can find out more about about Jaeger settings set up using environment variables by looking 
+at link:https://github.com/jaegertracing/jaeger-client-java/blob/v1.2.0/jaeger-core/README.md[jaeger-client-java readme].
+
+For the `JAEGER_PASSWORD` environment variable, the password can be encoded using the securityUtility command.
+
+Depending on Jaeger’s sampling settings `JAEGER_SAMPLER_TYPE` and `JAEGER_SAMPLER_PARAM`, 
+Jaeger may not report every spans created by the applications.
+
+For Zipkin, link:https://github.com/WASdev/sample.opentracing.zipkintracer[see here].
+
+Define your application in the server.xml:
+[source, xml]
+----
+<webApplication location="yourapp.war" contextRoot="/yourapp">
+    <!-- enable visibility to third party APIs -->
+    <classloader apiTypeVisibility="+third-party" />
+</webApplication>
+----
+
+Once you have hit some JAX-RS endpoints of your application, you should be able to find spans in the user interface of your tracing backend.
+
+More information:
+
+* link:https://github.com/eclipse/microprofile-opentracing[MicroProfile OpenTracting 2.0]
+
+[#jakarta]
+== Jakarta EE 9 Beta Features package
+
+The main change visible to developers in the Jakarta EE 9 planned release is the names of packages changing to accomodate the new `jakarta.*` namespace. In this Open Liberty beta, we have more Jakarta EE 9 features with their name change completed. We've also made excellent TCK progress with these features.
```

Comment:
Remove the sentence about TCK progress here for now. I've asked Kevin if we've made notable progress this month. See what he says.

Reply:
done